src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const activeTabId = tabs.some(tab => tab.id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+
+  const choozeTabClick = (tab) => {
```

Comment:
it seems that you choose click
```suggestion
  const handleTabClick = (tab) => {
```